lib/jsdom/living/nodes/Document.webidl

```diff
@@ -13,20 +13,20 @@ interface Document : Node {
 
   readonly attribute DocumentType? doctype;
   readonly attribute Element? documentElement;
-  HTMLCollection getElementsByTagName(DOMString localName);
+  HTMLCollection getElementsByTagName(DOMString qualifiedName);
   HTMLCollection getElementsByTagNameNS(DOMString? namespace, DOMString localName);
   HTMLCollection getElementsByClassName(DOMString classNames);
 
-  [NewObject] Element createElement(DOMString localName);
-  [NewObject] Element createElementNS(DOMString? namespace, DOMString qualifiedName);
+  [CEReactions, NewObject] Element createElement(DOMString localName, optional ElementCreationOptions options);
```

Comment:
I don't know if we should add this until we actually support custom elements.

Reply:
@domenic I thought of this last night and concur 100%. Perhaps we can roll this into #1030? Seems to be a touch stale but great progress nonetheless. Can take the enhancements from here and push that PR forward with these changes if you recommend this to be a good strategy.